notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>reduce installs to only the packages used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Done.